integration/main.sh

```diff
@@ -33,34 +33,35 @@ case ${1} in
        if [[ ${DESKTOP_FILE} != '' ]]; then
```

Comment:
Exit with usage message if `${2}` is missing?
`DESKTOP_FILE=$( find ${CREW_PREFIX} -name ${CMD}.desktop | sed -n '1p' )`?

Reply:
Easier?  `if [ -n ${DESKTOP_FILE} ]; then`